<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditBook</class>
 <widget class="QDialog" name="EditBook">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>513</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>421</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Edit Book</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	background-color: #1C1C1C;
}

QLineEdit{
	background-color: #2F2F2F;
	border-radius: 3px;
	color: white;
	border-bottom: 1px;
	border-style: solid;
	border-color: #979797;
}

QLineEdit:focus{
	border-color: #60CDFF;
}

QPushButton
{
    color: white;
    font-size: 15px;
    background-color: #292929;
    border-radius: 5px;
    font-family: &quot;Noto Sans&quot;
}

QPushButton::hover{ 
    background-color: #2F2F2F;
}

QPushButton:pressed{
    border-width: 1px;
    border-style: solid;
	border-color: white;
}

QLabel
{
    color: white;
	font-size: 16px;
    font-family: &quot;Noto Sans&quot;;l
    border-radius: 5px;
}
</string>
  </property>
  <property name="inputMethodHints">
   <set>Qt::ImhExclusiveInputMask</set>
  </property>
  <widget class="QPushButton" name="btnEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>470</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </widget>
  <widget class="QLabel" name="BookName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Book Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtBookName">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtAuthor1Name">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>60</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Author1Name">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Author 1 Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="DOP">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Date of Publication</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtEdition">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>210</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Edition">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>165</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Edition</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtClassificationNo">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>260</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="ClassificationNo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>165</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Classification No</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtPages">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>310</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Pages">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>165</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="text">
    <string>No. of Pages</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtAuthor2Name">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>110</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Author2Name">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>165</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Author 2 Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="ReferenceNo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>165</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Date of Acquisition</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtDateOfAcquisition">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>360</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Comments">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>165</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Comments</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtComments">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>410</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtDOP">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>160</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnReset">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>470</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
